stubs/third_party_taint/wtforms_sources.pysa

```diff
@@ -0,0 +1,14 @@
+ModelQuery(
+  find = "attributes",
+  where = [
+    AnyOf(
+      parent.extends("wtforms.form.Form", is_transitive=True),
+      parent.extends("flaskwtf.form.FlaskForm", is_transitive=True),
+    ),
+  ],
+  model = [
+    AttributeModel(
```

Comment:
CC @onionymous it's be great if we had a `type_annotation` where clause in `AttributeModel` to be able to restrict this to only attributes that are of type `Field`

Reply:
That would be super cool!